README.rst

```diff
@@ -1,3 +1,23 @@
+.. image:: https://img.shields.io/pypi/v/waitress.svg
+    :target: https://pypi.python.org/pypi/waitress
+    :alt: waitress on PyPI
+
+.. image:: https://img.shields.io/pypi/l/waitress.svg
+    :target: https://pypi.python.org/pypi/waitress
+    :alt: waitress on PyPI
+
+.. image:: https://img.shields.io/pypi/wheel/waitress.svg
+    :target: https://pypi.python.org/pypi/waitress
+    :alt: waitress on PyPI
```

Comment:
I feel that a screen reader user would need to know where the link is headed. I guess I could hard-code the alt attribute to say that a wheel is available, but if it's ever no longer available for newer versions, it might become inconsistent with the image. I would rather the alt attribute describe where the link is going.